model/generate.xml

```diff
@@ -407,13 +407,13 @@
   </command>
   
   <command symbol="mnemonic-new" output="string" multipleX="true" category="WALLET" description="Create a mnemonic seed (BIP39) from entropy. WARNING: mnemonic should be created from properly generated entropy.">
-    <option name="language" type="language" description="The language identifier of the mnemonic dictionary to use. Options are 'en', 'es', 'ja', 'zh_Hans', 'zh_Hant' and 'any', defaults to 'en'." />
+    <option name="language" type="language" description="The language identifier of the mnemonic dictionary to use. Options are 'en', 'es', 'ja', 'it', 'fr', 'zh_Hans', 'zh_Hant' and 'any', defaults to 'en'." />
```

Comment:
You should add comments to the XML as a reminder to update the command help in the wiki.